.github/workflows/ci.yml

```diff
@@ -28,7 +31,40 @@ jobs:
         run: npm run check
 
       - name: npm Test
-        run: npm run test
+        run: npm run test -- --coverage
```

Comment:
I think there are some extra --

Reply:
Surprisingly, the -- is required for getting the coverage report!